compose/web/start-server.sh

```diff
@@ -4,7 +4,7 @@ if [ -z ${PORT+x} ];
 then
     # PORT unset, we presume this is local dev:
     python manage.py migrate
-    yarn serve
+    DJANGO_ALLOW_ASYNC_UNSAFE=true yarn serve
```

Comment:
Instead of setting this via the command line, can we set it in `base.py` based on `DEV`? I don't use the `start-server` script at all in local development.

Reply:
Ah, I didn't know that you didn't. Sure thing.